bin/git-summary

```diff
@@ -80,17 +80,18 @@ file_count() {
 
 dedup_by_email() {
     # in:
-    # 27  luo zexuan <luozexuan@xxx.com>
+    # 27  luo zexuan <LuoZexuan@xxx.com>
     #  7  罗泽轩 <luozexuan@xxx.com>
     # out:
     # 34 luo zexuan
     LC_ALL=C awk '
     {
         sum += $1
-        if ($NF in emails) {
-            emails[$NF] += $1
+        lastField = tolower($NF)
+        if (lastField in emails) {
+            emails[lastField] += $1
         } else {
-            email = $NF
+            email = lastField
```

Comment:
Style: use underscore style variable

Reply:
Fixed. Perhaps the coding style could be mentioned in CONTRIBUTING.md.